src/bot/hashtagHandlers/handleDoneHashtag.ts

```diff
@@ -9,10 +12,28 @@ const isTrustedGroup = (chatId: number) => {
   return Object.values(chatIds).includes(chatId);
 };
 
+const createDoneRecords = (ctx: Context): TaskDoneRecord[] => {
+  if (!('message' in ctx.update) || !('text' in ctx.update.message)) return;
+  const { from, chat, text, date } = ctx.update.message;
+  const textsBeforeDone = getTextsBeforeDones(text);
+
+  return textsBeforeDone.map((textBeforeDone) => ({
+    groupId: chat.id,
+    groupName: 'title' in chat ? chat.title : null,
+    userId: from.id,
+    userName: from.first_name + (from.last_name ? ` ${from.last_name}` : ``),
+    textBeforeDone,
+    timestamp: new Date(date * 1000),
+    messageHash: `${Math.abs(from.id)}+${date}+${textBeforeDone}`,
+  }));
+};
+
 export const handleDoneHashtag = (bot: Telegraf) => {
   bot.hashtag('done', (ctx) => {
-    if (isTrustedGroup(ctx.message.chat.id)) {
-      ctx.reply(JSON.stringify(ctx.message));
+    console.log(ctx.message);
+    if (true || isTrustedGroup(ctx.message.chat.id)) {
```

Comment:
`true ||` - looks like some leftovers :). You can use `PROCESS_ENV.IS_DEVELOPMENT` instead